src/Symfony/Component/HttpKernel/Debug/TraceableEventDispatcher.php

```diff
@@ -61,7 +61,7 @@ protected function preDispatch($eventName, Event $event)
     protected function postDispatch($eventName, Event $event)
     {
         switch ($eventName) {
-            case KernelEvents::CONTROLLER:
+            case KernelEvents::CONTROLLER_ARGUMENTS:
```

Comment:
you're removing the stopwatch start on `controller` events then, to apply it only on `controller_arguments` events ?